q2_phylogeny/tests/test_fasttree.py

```diff
@@ -68,6 +68,48 @@ def test_fasttree_n_threads(self):
         self.assertEqual(tip_names, ['seq1', 'seq2'])
 
 
+# We are using pytest vs. unittest for this test
+# The capfd arg below is a built-in pytest fixture that allows for
+# captured stderr from all subprocesses that directly write
+# to operating system level output
+def test_fasttree_num_threads(capfd):
+    # Rather than providing an actual filepath, we just run a 'help' command
+    # Ideally, we supply --help in this param, but not available in FastTree
+    # So we are using -expert instead, so that FastTree doesn't fail
+    # ######################
+    # Output preview/example:
+    # ######################
+    # Detailed usage for FastTree 2.1.10 Double precision (No SSE3):
+    # FastTree [-nt] [-n 100] [-quote] [-pseudo | -pseudo 1.0]
+    #            [-boot 1000 | -nosupport]
+    #            [-intree starting_trees_file | -intree1 starting_tree_file]
+    #            [-quiet | -nopr]
+    #            [-nni 10] [-spr 2] [-noml | -mllen | -mlnni 10]
+    #            [-mlacc 2] [-cat 20 | -nocat] [-gamma]
+    #            [-slow | -fastest] [-2nd | -no2nd] [-slownni] [-seed 1253]
+    #            [-top | -notop] [-topm 1.0 [-close 0.75] [-refresh 0.8]]
+    #            [-matrix Matrix | -nomatrix] [-nj | -bionj]
+    #            [-lg] [-wag] [-nt] [-gtr] [-gtrrates ac ag at cg ct gt]
+    #            [-gtrfreq A C G T]
+    #            [ -constraints constraintAlignment
+    #            [ -constraintWeight 100.0 ] ]
+    #            [-log logfile]
+    #          [ alignment_file ]
+    #         [ -out output_newick_file | > newick_tree]
+    fasttree('-expert', n_threads=1)
+    captured = capfd.readouterr()
+    assert 'FastTree' in captured.err
```

Comment:
I think this test should be

```suggestion
    assert 'OpenMP' not in captured.err
```